plugins/outputs/azure_data_explorer/azure_data_explorer_test.go

```diff
@@ -18,217 +17,138 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const createTableCommandExpected = `.create-merge table ['%s']  (['fields']:dynamic, ['name']:string, ['tags']:dynamic, ['timestamp']:datetime);`
-const createTableMappingCommandExpected = `.create-or-alter table ['%s'] ingestion json mapping '%s_mapping' '[{"column":"fields", "Properties":{"Path":"$[\'fields\']"}},{"column":"name", "Properties":{"Path":"$[\'name\']"}},{"column":"tags", "Properties":{"Path":"$[\'tags\']"}},{"column":"timestamp", "Properties":{"Path":"$[\'timestamp\']"}}]'`
-
 func TestWrite(t *testing.T) {
+	metricName := "test1"
+	mockClient := kusto.NewMockClient()
+	expectedResultMap := map[string]string{metricName: `{"fields":{"value":1},"name":"test1","tags":{"tag1":"value1"},"timestamp":1257894000}`}
+	mockMetrics := testutil.MockMetrics()
+	// Multi tables
+	mockMetrics2 := testutil.TestMetric(1.0, "test2")
+	mockMetrics3 := testutil.TestMetric(2.0, "test3")
+	mockMetricsMulti := make([]telegraf.Metric, 2)
+	mockMetricsMulti[0] = mockMetrics2
+	mockMetricsMulti[1] = mockMetrics3
+	expectedResultMap2 := map[string]string{"test2": `{"fields":{"value":1.0},"name":"test2","tags":{"tag1":"value1"},"timestamp":1257894000}`, "test3": `{"fields":{"value":2.0},"name":"test3","tags":{"tag1":"value1"},"timestamp":1257894000}`}
```

Comment:
Can we please change the formatting here so it is easier to see what is going on?
```suggestion
	mockClient := kusto.NewMockClient()
	expectedResultMap := map[string]string{
		"test1": `{"fields":{"value":1},"name":"test1","tags":{"tag1":"value1"},"timestamp":1257894000}`},
	}
	mockMetrics := testutil.MockMetrics()
	// Multi tables
	mockMetricsMulti := []telegraf.Metric{
		testutil.TestMetric(1.0, "test2"),
		testutil.TestMetric(2.0, "test3"),
	}
	expectedResultMap2 := map[string]string{
		"test2": `{"fields":{"value":1.0},"name":"test2","tags":{"tag1":"value1"},"timestamp":1257894000}`,
		"test3": `{"fields":{"value":2.0},"name":"test3","tags":{"tag1":"value1"},"timestamp":1257894000}`,
	}
```